contracts/Pool.sol

```diff
@@ -38,62 +38,101 @@ contract Pool is IPool {
     /*
      * List of audit reports
      */
-
     AuditorReports.Reports internal audits;
 
     address private governer;
-    
 
-    /*
-     * Map to keep track of how much a user has deposited to and withdrawn from the Pool
-     */
-    mapping(address => uint256) userBalance;
-    
     modifier onlyGoverner {
         require(msg.sender == governer, "Only Governence allowed operation");
         _;
     }
-
+    
     modifier requireUserIsAdmin(address _address) {
         require(isAdmin[_address] == true,
             "User is not authorized to perform operation because they are not an Admin.");
         _;
     }
 
+    modifier requireDepositOrWithdrawMoreThanZero(uint256 _amount) {
+        require(_amount > 0, "Must deposit or withdraw more than zero.");
+        _;
+    }
+
     event AddedAdmin(address _address);
-    
+
     event RemovedAdmin(address _adddress);
     
     event AddedAuditReport(AuditorReports.Audit _audit);
 
+    event DepositedUSDC(address _address, uint256 usdcAmount);
+
+    event WithdrewKOLO(address _address, uint256 kTokenAmount);
+
     constructor (
         string memory _name,
         address _governer,
-        address[] memory _admins
-        ){
-
+        address[] memory _admins) {
         name = _name;
         governer = _governer;
-        
+
         // Deep copy
         for (uint i = 0; i < _admins.length; i++) {
-            if (!isAdmin[_admins[i]]) 
-                isAdmin[_admins[i]] = true;
-                totalAdmins += 1;
+            isAdmin[_admins[i]] = true;
```

Comment:
Why remove the check? It prevents double counting inputs with repeated admin adresses.

Reply:
This was done by accident, thanks for catching it